web-server/views/users/profile.jade

```diff
@@ -0,0 +1,25 @@
+extends ../layout
+
+block content
+
+  .row
+    .col-md-4
+    .col-md-4.profile-header
+      h1= user.name
+      p.profile-links
+        a(href="#{user.profileUrl}" target="_blank") @#{user.handle}
+        a(href="https://echo.learnersguild.org/users/#{user.handle}" target="_blank") Echo
+        a(href="https://echo.learnersguild.org" target="_blank") Guide
+    .col-md-4  
+  .row.profile-content
+    .col-md-2
+    .col-md-4
+      img.img-thumbnail.curriculum-profile-photo(src="#{user.avatarUrl}" alt="avatar")
+    .col-md-4.user-profile-info
+      h5
+        | Email: 
+        a(href="mailto:#{user.email}" target="_blank") porchewellness@gmail.com
+      h5=`Phone: ${user.phone}`
+      h5 Join Date: #{moment(user.createdAt).format("MMMM Do YYYY")}
+      h5 Updated: #{moment(user.updatedAt).format("MMMM Do YYYY")}
```

Comment:
change these to the new data helpers you'll find on master and can use after you rebase